docs/quick-start.md

```diff
@@ -61,6 +61,25 @@ def --env y [...args] {
 	}
 	rm -fp $tmp
 }
+```
+
+  </TabItem>
+  <TabItem value="elvish" label="Elvish">
+
+```elvish
+edit:add-var y~ {|@argv|
+	use str
+	use os
+	use file
```

Comment:
Is it possible to merge them into one line, like `use str, os, file`?

Reply:
You can't combine into one line. For this one I'm actually a bit torn between copy-ability and idiomatic usage.  Realistically you would have the `use` imports at the top of the file, I could pull out the imports, eg:

```elvish
use str
use os
use file
edit:add-var y~ {|@argv|
  ...
}
```

Reply:
Or I could assume other elvish users would know they had to import and omit the `use` statements

Reply:
Ah I see, that's fine, let's just keep it as is. Thanks for the explanation!